packages/main/src/Carousel.ts

```diff
@@ -152,28 +152,17 @@ class Carousel extends UI5Element {
 	cyclic!: boolean;
 
 	/**
-	 * Defines the number of items per page on small size (up to 640px). One item per page shown by default.
-	 * @default 1
+	 * Defines the number of items per page depending on the page width.
+	 * 'S' for screens smaller than 600 pixels.
+	 * 'M' for screens greater than or equal to 600 pixels and smaller than 1024 pixels.
+	 * 'L' for screens greater than or equal to 1024 pixels and smaller than 1440 pixels.
+	 * 'XL' for screens greater than or equal to 1440 pixels.
+	 * One item per page is shown by default.
+	 * @default "S1 M1 L1 XL1"
```

Comment:
Leave empty line for new row. Add "-" for bullet points.
So make it like this:
/**
	 * Defines the number of items per page depending on the page width.
	 * 
	 * - 'S' for screens smaller than 600 pixels.
	 * - 'M' for screens greater than or equal to 600 pixels and smaller than 1024 pixels.
	 * - 'L' for screens greater than or equal to 1024 pixels and smaller than 1440 pixels.
	 * - 'XL' for screens greater than or equal to 1440 pixels.
	 * 
	 * One item per page is shown by default.
	 * @default "S1 M1 L1 XL1"
	 * @public
	 */